content/guides/component-testing/slots-vue.md

```diff
@@ -39,21 +39,23 @@ previous sections, we'll start simple.
 ```js
 it('renders the modal content', () => {
   cy.mount(Modal, { slots: { default: () => 'Content' } })
-    .get(modalSelector)
-    .should('have.text', 'Content')
+
+  cy.get(modalSelector).should('have.text', 'Content')
```

Comment:
All of these component testing pages have gone away, and most of these examples (if they still exist at all) are now either in the quickstart or example guides for each framework.

Reply:
Thanks for the heads up. Merged in latest v12 and fixed the merge conflicts.